### DISCUSSION TITLE
Who wants a search function on the select node menu?

### DISCUSSION THREAD

**Discussion by BrechtCorbeel** [comfyanonymous/ComfyUI#3886] [Ideas]
Unless it Already exists if someone can point at where the menu code is I'll See if I can get it working and push this in if accepted.

**Comment by bigcat88** [comment]
_I want_

**Reply by TwoPintOhh** [reply]
Yes that would be great

**Comment by BrechtCorbeel** [comment]
anybody found or made any extension for it?

**Reply by bigcat88** [reply]
I think this will be included in the new ComfyUI frontend that will come in August

**Reply by ltdrdata** [reply]
@huchenlei ping
